LICENSE.APL.txt

```diff
@@ -0,0 +1,37 @@
+==========================================================
+Licensing Terms for he PyInstaller community runtime hooks
+==========================================================
+
+The PyInstaller community **runtime** hooks are licensed under the terms of the
+Apache 2.0 Software License as published by the Apache Software Foundation.
+
+Please note that this license only applies to files which have the following header:
+
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the Apache License 2.0
+#
+# The full license is available in LICENSE.APL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ------------------------------------------------------------------
+
+
+Apache License 2.0
+++++++++++++++++++
```

Comment:
I'm curious: While you added the complete text of the GPL, but not of the APL. Any reason?

Reply:
https://github.com/pyinstaller/pyinstaller-hooks-contrib/pull/2#discussion_r408765869